libmscore/input.h

```diff
@@ -79,8 +88,11 @@ class InputState {
       int string() const                  { return _string;             }
       void setString(int val)             { _string = val;              }
 
-      bool repitchMode() const            { return _repitchMode;    }
-      void setRepitchMode(bool val)       { _repitchMode = val;     }
+      bool repitchMode() const            { return _noteEntryMethod == NoteEntryMethod::REPITCH;    }
+      void setRepitchMode(bool val)       { if(val)
+                                                setNoteEntryMethod(NoteEntryMethod::REPITCH);
+                                            else
+                                                setNoteEntryMethod(NoteEntryMethod::STEPTIME);}
```

Comment:
Or I could replace the `noteEntryMethod()` function:
```C++
NoteEntryMethods noteEntryMethod() { return _noteEntryMethod; }
```
With a new one:

```C++
bool isNoteEntryMethod(NoteEntryMethods method) {
      if (noteEntryMode() && _noteEntryMethod == method)
            return true;
      else
            return false;
      }
```

And use it like this:
```C++
// Test for Repitch mode
if (isNoteEntryMethod(NoteEntryMethods::REPITCH)) {
            qDebug("In Repitch mode!");
      }
```